src/Farmer/Builders/Builders.DeploymentScript.fs

```diff
@@ -126,8 +136,8 @@ type DeploymentScriptBuilder() =
     /// Time to retain the container instance that runs the script - 1 to 26 hours.
     [<CustomOperation "retention_interval">]
     member _.RetentionInterval(state:DeploymentScriptConfig, retentionInterval) =
-        let maxRetention = min retentionInterval 30<Hours>
-        { state with RetentionInterval = Some (TimeSpan.FromHours (float maxRetention)) }
+        let maxRetention = min retentionInterval 26<Hours>
```

Comment:
More and more I'm starting to think that we should get rid of these implicit "fixing bad data" issues and simply fail fast with e.g. a failwith - it's clearer to the caller what's gone wrong (and the error message can / should make it clear how to fix it), rather than them assume 100 hours has worked and then realised later on that 26 hours is the max. What do you think?